sass/base/_compass-icons.scss

```diff
@@ -192,7 +203,6 @@
 .icon-bell-off-outline:before { content: '\fa90'; } /* '敖' */
 .icon-chevron-down-circle-outline:before { content: '\fb0c'; } /* '﬌' */
 .icon-format-letter-case:before { content: '\fb19'; } /* '﬙' */
-.icon-account-group-outline:before { content: '\fb34'; } /* 'הּ' */
```

Comment:
@michaelgamble same with this one. Looks like `icon-account-group-outline` has been removed

Reply:
Looks like this one never come over ... I checked the svg folder on my initial commit and it's not there. Maybe I/we forgot to add it in here?

Reply:
Maybe. That would be strange though since it's in our original files.

Reply:
Actually, I'm wrong. `icon-account-group-outline` is not part of our original files after all. I'm not sure how it was in there in the first place.